compiler/src/syntax.ml

```diff
@@ -214,7 +214,15 @@ type align = [`Align | `NoAlign]
 
 type plvals = annotations L.located option * plvalue list
 
-type vardecls = pstotype * pident list
+
+type vardecl = InitVarDecl of pident * pexpr | NotInitVarDecl of pident
```

Comment:
I think the definition
type vardecl = pident * pexpr option
will be more convenient.
You can do the change if you want.